### PR TITLE
More translation

### PR DESCRIPTION
**Pull request by SondreNjaastad**

### CODE REVIEW

File: src/UI/UIDesktop.js [hunk 1]
```diff
@@ -1197,18 +1197,19 @@ $(document).on('click', '.user-options-menu-btn', async function(e){
                     // see if there are any open windows, if yes notify user
                     if($('.window-app').length > 0){
                         const alert_resp = await UIAlert({
-                            message: `<p>You have open apps. Are you sure you want to log out?</p>`,
+                            message: `<p></p>`,
```
Comment (jelveh) [inline src/UI/UIDesktop.js]: I think this was removed unintentionally?

File: src/UI/UIItem.js [hunk 2]
```diff
@@ -1245,14 +1245,14 @@ function UIItem(options){
                     html: i18n('delete_permanently'),
                     onClick: async function(){
                         const alert_resp = await UIAlert({
-                            message: `Are you sure you want to permanently delete this item?`,
+                            message: ` you want to permanently delete this item?`,
```
Comment (jelveh) [inline src/UI/UIItem.js]: This one looks wrong.

**Review by jelveh — CHANGES_REQUESTED** [review]

File: src/UI/UIDesktop.js (hunk 1 above)
Comment (SondreNjaastad) [reply src/UI/UIDesktop.js]: Yes, it should have bern replaced with a i18n replacement. Will fix

File: src/UI/UIItem.js (hunk 2 above)
Comment (SondreNjaastad) [reply src/UI/UIItem.js]: It is wrong , for sure 😅

**Comment by jelveh** [comment]
Another awesome PR, thank you!

**Pull request merged**